code/modules/unit_tests/purchase_reference_test.dm

```diff
@@ -12,17 +12,3 @@
 			Fail("uplink reference [uplink_ref] is used multiple times")
 		uplink_refs += uplink_ref
 
-/datum/unit_test/spellbook_refs/Run()
```

Comment:
do we still want a unit_test to make sure we don't have duplicate names in the spellbook or is that probably not gonna be an actual issue?

Reply:
Can add something if we want

Reply:
not really a big deal